pages/docs/community/tooling.md

```diff
@@ -116,6 +125,17 @@ The following is a list of tools that validate AsyncAPI documents.
 
 ---
 
+# Compare tools {#diff}
+
+The following is a list of tools that compare AsyncAPI documents.
+
+| Link           | Description    | Language/Framework |
+| :------------- | :------------- | :----------------- |
+| [diff](https://github.com/asyncapi/diff) | Diff is a library that compares two AsyncAPI Documents and provides information about the differences by pointing out explicitly information like breaking changes. | Javascript
+| [api-smart-diff](https://github.com/udamir/api-smart-diff) | It allows you to compare two API specifications (OpenAPI, AsyncAPI, JsonSchema). Thanks to [udamir](https://github.com/udamir). | Javascript
```

Comment:
<img width="500" alt="Screen Shot 2022-05-19 at 10 49 40 AM" src="https://user-images.githubusercontent.com/19964402/169365276-ce8070c4-62c7-48b4-8d6d-bc82bd1789f3.png">

This reads like a mistake.. why do we mention **two** API specifications... but then add a list of 3 items (OpenAPI, AsyncAPI, JsonSchema)?

Reply:
done